modules/shorten.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file shorten.js
+ * @description Module for shortening links using "acmurl" (Rebrandly API).
+ * @author Storm_FireFox1
+ */
+
+global.config = require('../config.json')
+const rp = require('request-promise')
+const url = require('url').URL
+const https = require('https')
+
+module.exports = {
+
+    // List of available commands in the module
+    commands: [
+      'shorten'
+    ],
+
+    // A description of this module
+    description: "Interacts with 'acmurl' in various ways.",
+
+    // An image representing the module
+    thumbnail: '',
+
+    'shorten': {
+      usage: '!shorten <shortlink> <longurl> [description]',
+      description: "Shortens the provided URL into a 'acmurl' link.",
+      method: (client, message, args) => {
+        if (!message.member.roles.some(r => r.name === "Board")) {
+          message.channel.send("You must be a Board member to use this command!");
+          return;
+        }
+        // URL Validator taken straight from Stack Overflow
+        //
+        // (https://stackoverflow.com/questions/30931079/validating-a-url-in-node-js)
+        //
+        // Needed to check if arguments provided are actually URL's for
+        // Rebrandly.
+        const validURL = (s) => {
+          try {
+            new URL(s);
+            return true;
+          } catch (err) {
+            return false;
+          }
+        };
+
+        // Check if there are not two arguments
+        if (args.length < 2) {
+          message.channel.send("You must provide both the long link and the short link!");
+          return;
+        } else if (!validURL(args[1])) { // check for valid URL
+          message.channel.send("The long link must be a valid URL!");
+          return;
+        }
+
+        // Options for request
+        const addLinkOptions = {
+          uri: 'https://api.rebrandly.com/v1/links',
+          method: 'POST',
+          headers: {
+            'apikey': config.rebrandlyAPIKey,
+            'Content-Type': 'application/json'
+          },
+          body: {
+            title: args[2] || 'Discord Bot - ' + args[0], // optional argument or slashtag
```

Comment:
nit: assign the `args` elements to variables so these expressions are clearer, e.g.
```javascript
const [shorturl, longurl] = args;

const shorturl = args[0];
const longurl = args[1];
```

Reply:
I've updated this part, you can check 0ea150e.

Reply:
I've made expressions clearer, see 0ea150e.